src/content/docs/en/reference/api-reference.mdx

```diff
@@ -807,29 +924,40 @@ The `CollectionEntry<TCollectionName>` type is an object with the following valu
 
 #### `id`
 
-**Example Type:** `'entry-1.md' | 'entry-2.md' | ...`
+**Example Type for "content" collections:** `'entry-1.md' | 'entry-2.md' | ...`  
+**Example Type for "data" collections:** `'author-1' | 'author-2' | ...`
 
-A unique ID using the file path relative to `src/content/[collection]`. Enumerates all possible string values based on the collection entry file paths.
+A unique ID using the file path relative to `src/content/[collection]`. Enumerates all possible string values based on the collection entry file paths. Note that collections [defined as `type: 'content'`](#type) include the file extension in their ID, while collections defined as `type: 'data'` do not.
 
-#### `slug`
+#### `collection`
 
-**Example Type:** `'entry-1' | 'entry-2' | ...`
+**Example Type:** `'blog' | 'authors' | ...`
 
-A URL-ready slug. Defaults to the `id` without the file extension, but can be overridden by setting [the `slug` property](/en/guides/content-collections/#defining-custom-slugs) in a file's frontmatter.
+The name of the entry's collection.
 
 #### `data`
 
 **Type:** `CollectionSchema<TCollectionName>`
 
 An object of frontmatter properties inferred from your collection schema ([see `defineCollection()` reference](#definecollection)). Defaults to `any` if no schema is configured.
 
-#### `body`
+#### Content-specific properties
```

Comment:
Not entirely sure what to do here, but it's kind of weird that `slug`, `body` and `render` now end up a heading size smaller than the other properties. I wonder if there's a way to identify these as only available for `type:content` without another heading. Is it too terribly messy/ugly to include for each one of these, maybe under **Type**, "available for every collection `type`" or "available for `type: content` collections only"  ??

Reply:
I spent _so long_ trying this, and it was much more confusing with the "for type whatever" callouts everywhere. I agree the heading levels are annoying though this made it much clearer which were content-specific 😓 If you have a better refactor, push it up! This is the best I could come up with

Reply:
OK, for now let's embrace the weirdness! No need to get this right on the first try. I have no issues with the content, so maybe the proper formatting will just reveal itself to us, eventually.

Reply:
Took a stab at refactoring to "Available for" callouts instead. Let me know if that's any better!

<img width="661" alt="image" src="https://github.com/withastro/docs/assets/51384119/61eba2f2-4519-4394-86c5-7ebf79ac10c1">

Reply:
I do personally like that better, because I like having all the properties in one place (rather than people needing to know where to go for certain types of properties before even being able to read about them). As the list of `type`s grows (?) we can always consider using "all types" for some properties, and "content only" for others.